.builder/actions/build_samples.py

```diff
@@ -11,9 +11,12 @@ def run(self, env):
 
         steps = []
         samples = [
-            'samples/mqtt/basic_pub_sub',
-            'samples/mqtt/pkcs11_pub_sub',
-            'samples/mqtt/raw_pub_sub',
+            'samples/pub_sub/basic_pub_sub',
+            'samples/mqtt/basic_connect',
+            'samples/mqtt/pkcs11_connect',
+            'samples/mqtt/raw_connect',
+            'samples/mqtt/websocket_connect',
+            'samples/mqtt/x509_connect',
```

Comment:
we might want a different name for the "x509_connect", since the "basic_connect" is also technically using X.509 certificates.

The difference is that "basic_connect" sample uses X.509 certificates during the TLS handshake.

While the "x509_connect" sample uses basic TLS (only the server's identity is authenticated) during the TLS handshake, but then an X.509 certificate is used to prove the client's identity during the websocket handshake. Maybe call this "websocket_x509_connect"? Or "x509_cred_provider_connect"?

Run it by Bret, he knows more about credentials providers

Reply:
It's wordy, but x509_credentials_provider_connect is accurate.

Reply:
I'll change it to "x509_credentials_provider_connect" 👍